_maps/virtual_domains/ash_drake.dmm

```diff
@@ -21,7 +21,7 @@
 /obj/machinery/light/small/blacklight/directional/south,
 /obj/effect/baseturf_helper/virtual_domain,
 /turf/open/misc/asteroid/basalt/lava_land_surface/no_ruins,
-/area/virtual_domain/powered)
```

Comment:
removing these in the future, if you're wondering. there will only be two types: base and fullbright